packages/dso-toolkit/src/components/card-container/card-container.args.ts

```diff
@@ -27,16 +27,15 @@ export const cardContainerArgs: Omit<CardContainerArgs, "mode"> = {
     {
       label: "Begrippen uit de Omgevingswet",
       href: "#",
+      active: false,
       selectable: true,
       interactions: [],
-      image: undefined,
-      imageAlt: undefined,
-      imageShape: "normal",
-      clickable: false,
+      dsoCardClick() {},
```

Comment:
En volledigheidshalve, ook deze event listener kan weg zodra de eventlistener in het model optioneel is gemaakt

Reply:
En dat kan dus niet omdat het hier gaat om `cards: CardArgs[]` en in `CardArgs` is `dsoCardClick` niet optioneel:

```
export interface CardArgs {
  label: string;
  href: string;
  active: boolean;
  mode?: AnchorArgs["mode"];
  selectable: boolean;
  interactions: Array<Button | Label | Toggletip<never> | SlideToggle>;
  dsoCardClick: HandlerFunction;
}
```

En eerder heb ik daar juist de optionaliteit weggehaald ... 🤷‍♂️

Reply:
In overleg met Eric laten we het staan zoals het nu is.